README.md

```diff
@@ -14,7 +18,7 @@ Docmaker will try to find `docmaker.yaml` in the current directory or above and
 
 Via a config file called `docmaker.yaml` you can specify pages to render, which data to load and which layout HTML file the document should use.
 
-### Example
+### Example:
```

Comment:
I don't know why, but I just don't like colons in headers.

```suggestion
### Example
```

Reply:
Agreed.